src/env.js

```diff
@@ -28,6 +30,37 @@ const env = createEnv({
     DOCKER_PORT: portSchema,
     DEMO_MODE: z.string().optional(),
     HOSTNAME: z.string().optional(),
+
+    // Authentication
+    AUTH_PROVIDER: z.string().default('credentials').transform(providers => providers.replaceAll(' ', '').split(',')),
+    // LDAP
+    ...(authProviders.includes('ldap')
+      ? {
+          AUTH_LDAP_URI: z.string().url(),
+          AUTH_LDAP_BIND_DN: z.string(),
+          AUTH_LDAP_BIND_PASSWORD: z.string(),
+          AUTH_LDAP_BASE: z.string(),
+          AUTH_LDAP_USERNAME_ATTRIBUTE: z.string().default('uid'),
+          AUTH_LDAP_GROUP_CLASS: z.string().default('groupOfUniqueNames'),
+          AUTH_LDAP_GROUP_MEMBER_ATTRIBUTE: z.string().default('member'),
+          AUTH_LDAP_GROUP_MEMBER_USER_ATTRIBUTE: z.string().default('dn'),
+          AUTH_LDAP_ADMIN_GROUP: z.string().default('admin'),
+          AUTH_LDAP_OWNER_GROUP: z.string().default('admin'),
+        }
+      : {}),
+    // OIDC
+    ...(authProviders.includes('oidc')
+      ? {
+          AUTH_OIDC_CLIENT_ID: z.string(),
+          AUTH_OIDC_CLIENT_SECRET: z.string(),
+          AUTH_OIDC_URI: z.string().url(),
+          // Custom Display name, defaults to OIDC
+          AUTH_OIDC_CLIENT_NAME: z.string().default('OIDC'),
```

Comment:
I was able to set ``AUTH_OIDC_CLIENT_ID`` even with unset ``AUTH_PROVIDER``. Is this intentional?